apps/whale-api/src/module.indexer/model/dftx/place.auction.bid.ts

```diff
@@ -1,17 +1,20 @@
 import { DfTxIndexer, DfTxTransaction } from './_abstract'
 import { PlaceAuctionBid, CPlaceAuctionBid } from '@defichain/jellyfish-transaction'
 import { RawBlock } from '../_abstract'
-import { Injectable, Logger } from '@nestjs/common'
+import { Inject, Injectable, Logger } from '@nestjs/common'
 import { VaultAuctionHistoryMapper } from '../../../module.model/vault.auction.batch.history'
-import { toBuffer } from '@defichain/jellyfish-transaction/dist/script/_buffer'
+import { NetworkName } from '@defichain/jellyfish-network'
 import { HexEncoder } from '../../../module.model/_hex.encoder'
+import { fromScriptHex } from '@defichain/jellyfish-address'
+import { toBuffer } from '@defichain/jellyfish-transaction/dist/script/_buffer'
 
 @Injectable()
 export class PlaceAuctionBidIndexer extends DfTxIndexer<PlaceAuctionBid> {
   OP_CODE: number = CPlaceAuctionBid.OP_CODE
   private readonly logger = new Logger(PlaceAuctionBidIndexer.name)
 
   constructor (
+    @Inject('NETWORK') private readonly network: NetworkName,
```

Comment:
For the indexer we should keep it network agnostic, we shoukd decode and adapt it on the controller level instead of the indexer level.

Reply:
moved to controller level